src/kmer.jl

```diff
@@ -156,7 +156,8 @@ function Kmer{A,K,N}(itr) where {A,K,N}
     # Construct the head.
     head = zero(UInt64)
     @inbounds for i in 1:n_head
-        sym = convert(eltype(Kmer{A,K,N}), itr[i])
```

Comment:
We're not able to index into an iterator, but we can iterate it at the index that we want